components/Footer.js

```diff
@@ -75,29 +75,16 @@ export default function Footer (props) {
           <ul>
             <li>
               <NavLink
-                // FIXME: Update link
-                href='/'>
+                target='_blank'
+                href={`mailto:${process.env.SUPPORT_EMAIL}?subject=${process.env.SITE_TITLE} Support Request`}>
                 Help
               </NavLink>
             </li>
-            <li>
-              <NavLink
-                // FIXME: Update link
-                href='/'>
-                Request access to API
-              </NavLink>
-            </li>
           </ul>
         </div>
       </section>
       <section className='ft-legal'>
         <ul className='ft-legal-list'>
-          <li>
-            <NavLink
-              href={termsPath}>
-              Terms &amp; Conditions
```

Comment:
A terms an conditions link is shown in the user sign up page, is that still the intent?

Reply:
Nope, good catch!

Reply:
Fixed in c21b67c